pystac_client/mixins.py

```diff
@@ -32,14 +32,24 @@ def _get_href(self, rel: str, link: Optional[pystac.Link], endpoint: str) -> str
 class QueryablesMixin(BaseMixin):
     """Mixin for adding support for /queryables endpoint"""
 
-    def get_queryables(self) -> Dict[str, Any]:
-        """Return all queryables.
+    def get_queryables(self, *collections: Optional[str]) -> Dict[str, Any]:
+        """Return all queryables, or limit to those of specified collections.
```

Comment:
```suggestion
        """Return all queryables, or limit to those of specified collections.
        
        Queryables from multiple collections are unioned together, except in the case when the same queryable key has a different definition, in which case that key is dropped.
```

Suggested language to capture my suggestion in https://github.com/stac-utils/pystac-client/issues/505#issuecomment-1538297777.